tests/expr_and_series/drop_nulls_test.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from typing import Any
+
+import narwhals as nw
+from tests.utils import compare_dicts
+
+
+def test_drop_nulls(constructor: Any) -> None:
+    data = {
+        "A": [1, 2, None, 4],
+        "B": [5, 6, 7, 8],
+        "C": [None, None, None, None],
+        "D": [9, 10, 11, 12],
+    }
+
+    df = nw.from_native(constructor(data))
+
+    result_a = df.select(nw.col("A").drop_nulls())
+    result_b = df.select(nw.col("B").drop_nulls())
```

Comment:
let's change one to the series implementation to make sure we cover that too
```suggestion
    result_b = df.select(df["B"].drop_nulls())
```